app/models/post.rb

```diff
@@ -119,11 +122,13 @@ class Post < ApplicationRecord
   validates :title, presence: true, length: { minimum: 5, maximum: 75 }
   validates :code, presence: true, uniqueness: { case_sensitive: false }, length: { minimum: 5, maximum: 6 }, format: { with: /\A[A-Za-z0-9]+\z/, message: "is invalid. Only letters and numbers are allowed." }
   validates :nice_url, uniqueness: true, allow_blank: true, length: { minimum: 7, maximum: 20 }, format: { with: /\A[a-z0-9-]+\z/, message: "is invalid. Only lowercase letter, numbers, and dashes are allowed." }
+  validates :description, length: { maximum: 100000 }
```

Comment:
It would be nice if we could DRY these limit values out so that if we need to change them in the future, it won't be necessary to comb the entire codebase. See [Ruby on Rails: Where to define global constants?](https://stackoverflow.com/questions/4110866/ruby-on-rails-where-to-define-global-constants) for some suggestions on how to hoist this value up into a One True value.